### PR TITLE
(Ready to be merged) Elk configurations final

### PR DESCRIPTION
**Pull request by genagain**
This pull request installs the filebeat service on the compute and controller nodes. Once filebeat is running on a node, the any of the log files can be forwarded to an server running the ELK stack. Once the logs from different nodes are being aggregated by the server running the ELK stack, log files can be searched and visualized really quickly

### CODE REVIEW

**Comment by genagain** [comment]
Because there were too many commits on my [last branch](https://github.com/CCI-MOC/kilo-puppet/pull/40). I created this new branch.

**Comment by genagain** [comment]
Also @lokI8 said that my last branch looks good provided that I implemented his feedback, which I did in this branch.

File: quickstack/manifests/compute_common.pp [hunk 1]
```diff
@@ -409,11 +409,27 @@
...
+      }
+    },
+    logging => {
+      'level' => "info"
+    }
+  }
+
+  filebeat::prospector { 'generic':
+      paths => ["/var/log/*.log", "/var/log/secure", "/var/log/messages", "/var/log/ceph/*", "/var/log/nova/*", "/var/log/neutron/*", "/var/log/openvswitch/*", "/var/log/cinder/", "/var/log/glance/", "/var/log/horizon/", "/var/log/httpd/", "/var/log/keystone/*"]
+  }
```
Comment (rahulait) [inline quickstack/manifests/compute_common.pp:431]: I guess it should be /var/log/cinder/\* and same with other folders. I added this in previous comment, but github removed the trailing *. You can fix this.

File: quickstack/manifests/controller_common.pp [hunk 2]
```diff
@@ -840,12 +841,28 @@
...
+      }
+    },
+    logging => {
+      'level' => "info"
+    }
+  }
+
+   filebeat::prospector { 'generic':
+      paths => ["/var/log/*.log", "/var/log/secure", "/var/log/messages", "/var/log/ceph/*", "/var/log/nova/*", "/var/log/neutron/*", "/var/log/openvswitch/*", "/var/log/cinder/", "/var/log/glance/", "/var/log/horizon/", "/var/log/httpd/", "/var/log/keystone/*"]
+    }
```
Comment (rahulait) [inline quickstack/manifests/controller_common.pp:859]: same as above

File: quickstack/manifests/compute_common.pp (hunk 1 above)
Comment (genagain) [inline quickstack/manifests/compute_common.pp:431]: I just changed it! Thank you for catching this.

**Comment by rahulait** [comment]
+1

**Comment by lokI8** [comment]
Looks good to me, +1 and merging =)

**Pull request merged**